### ISSUE TITLE
Nested persist

### ISSUE THREAD

**Issue by eseQ** [hilkeheremans/redux-persist-seamless-immutable#12]
Hello.
On using [nested persist](https://github.com/rt2zz/redux-persist#nested-persists) with a transform to seamless-immutable has a failure.
Transformer returns a plain js object. [Not seamless-immutable](https://github.com/hilkeheremans/redux-persist-seamless-immutable/blob/master/src/transformer.js#L33).
On root level all right.

**Comment by hilkeheremans** [comment]
Yes, that doesn't surprise me. I'm willing to look into it if you can include a minimal repro here?

**Comment by eseQ** [comment]
Yeah, sure.
Here is [a repro](https://codesandbox.io/s/lp21nn68jl?expanddevtools=1&fontsize=14).
Feel free to ask.

**Comment by dlindstrm** [comment]
@eseQ @hilkeheremans any progress? I'm also stuck at this problem...

**Comment by henrych4** [comment]
I have made some changes to the code so that it can support nested persist.
The syntax looks like:
```
{
  blacklistPerReducer: {
      reducerB: ['keyA', 'keyB.keyB1']
  }
}
```

Any comment or suggestion on this syntax? If it looks good then I can submit a pull request
